src/main/java/com/github/fppt/jedismock/operations/RO_zrem.java

```diff
@@ -0,0 +1,40 @@
+package com.github.fppt.jedismock.operations;
+
+
+import com.github.fppt.jedismock.server.Response;
+import com.github.fppt.jedismock.server.Slice;
+import com.github.fppt.jedismock.storage.RedisBase;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+
+import static com.github.fppt.jedismock.Utils.deserializeObject;
+import static com.github.fppt.jedismock.Utils.serializeObject;
+
+public class RO_zrem extends AbstractRedisOperation {
```

Comment:
Same thing here. Maybe we can abstract into a `RO_rem` class which `RO_zrem` and `RO_lrem` can share.